src/test/java/uk/gov/hmcts/reform/sendletter/services/LetterServiceTest.java

```diff
@@ -72,6 +73,15 @@ public void setUp() {
         failedCompletableFuture.completeExceptionally(new Exception("some exception"));
     }
 
+    @Test
+    public void should_return_letter_status_when_it_is_found_in_database() {
```

Comment:
We should also test the opposite case - throws exception when letter not found.

Reply:
it should. letter service is not yet implemented and nothing actually is capable in throwing an exception or whatsoever

Reply:
Fair enough